<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogIntrinsic</class>
 <widget class="QDialog" name="DialogIntrinsic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="TOGGLE">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>550</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BACK">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>550</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DELETE">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>550</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>180</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QListWidget" name="imageList">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>300</y>
     <width>231</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cameraTypeCombo">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>50</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="BANNER">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Intrinsic Calibration</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
